src/dashboard_links.py

```diff
@@ -0,0 +1,149 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+"""Tools for managing dashboard links from relations and charm config."""
+import logging
+from typing import List
+
+import yaml
+from charms.kubeflow_dashboard.v0.kubeflow_dashboard_links import (
+    DashboardLink,
+    dashboard_links_to_json,
+)
+
+logger = logging.getLogger(__name__)
+
+
+def aggregate_links(
+    links_from_relation: List[DashboardLink], additional_link_config: str, link_order_config: str
+):
```

Comment:
docstring missing here, same for the method below

Reply:
added!